src/main/controllers/auth_logout.ts

```
@@ -6,6 +6,7 @@ export default {
         request.logout({ keepSessionInfo: false }, ((error: any) => {
```

ESLint

[error] Delete `(`

(prettier/prettier)